mouse/record.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
We haven't discussed this yet, but it would be nice if we can ensure Python 3 compatibility. One step on that path is using the `__future__` module. A few things to note about it below.

* Only `from ... import ...` can be used with it.
* It must be the first line in your program.

Add the line `from __future__ import print_function`.